packages/react/src/components/DatePicker/DatePicker.js

```diff
@@ -413,6 +417,15 @@ export default class DatePicker extends Component {
           this.cal.close();
         }
       });
+      this.toInputField.addEventListener('keydown', e => {
+        if (e.which === 40) {
```

Comment:
should we include a refactor to use these [keyboard handlers](https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/internal/keyboard/keys.js)?

Reply:
Might as well: 

```js
import { match, keys } from '../../internal/keyboard';

if (match(event, keys.ArrowDown))
```